### PR TITLE
Blog Title Overflow Fix 

### LINKED ISSUE #274: title overflow
### What happened?

image
see image

### PR DESCRIPTION
**Pull request by stephaniePocci**
## **This PR fixes issue #274**
I was able to fix the blog listings so that there is no more text overflow from the titles. The titles now wrap, truncate, and place ellipsis. See screenshots below:
image
image
I'm unable to show the changes in vercel since I had to enable debug mode for localhost changes, so reviewers may need to clone my [fork](https://github.com/stephaniePocci/acmcsuf.com) locally to view the changes. Please feel free to let me know though if there is a way to enable the blogs to be viewed in vercel!

Also, I noticed there is another issue where the titles still do text overflow on the actual blog pages, so I will be making an issue for that soon.

### CODE REVIEW

File: src/routes/blog/index.svelte [hunk 1]
```diff
@@ -193,6 +193,13 @@
           }
         }
 
+        a > h2 {
+          // white-space: nowrap; // epepepe
```
Comment (charliettaylor) [inline src/routes/blog/index.svelte]: Remove commented code plox

Comment (stephaniePocci) [reply src/routes/blog/index.svelte]: o7 yessir

File: src/routes/blog/index.svelte [hunk 2]
```diff
@@ -193,6 +193,12 @@
           }
         }
 
+        a > h2 {
+          overflow: hidden;
+          text-overflow: ellipsis;
+          min-width: 0;
```
Comment (jaasonw) [inline src/routes/blog/index.svelte]: why min width here? default value for min-width is already 0 and doesnt seem to affect anything when i comment out

Comment (jaasonw) [inline src/routes/blog/index.svelte]: unless what you meant to do was a max width/height as this does not currently truncate
image

**Review by jaasonw — CHANGES_REQUESTED** [review]

File: src/routes/blog/index.svelte (hunk 2 above)
Comment (stephaniePocci) [reply src/routes/blog/index.svelte]: I yoinked it out

**Review by EthanThatOneKid — APPROVED** [review]
LGTM!

**Review by jaasonw — APPROVED** [review]
👍

**Comment by jaasonw** [comment]
> Also, I noticed there is another issue where the titles still do text overflow on the actual blog pages, so I will be making an issue for that soon.

is this fix not something that can just be directly applied to the blog pages?

**Review by k4rni — APPROVED** [review]
So proud of you! Great work:)

**Review by charliettaylor — APPROVED** [review]
May the flood of text not overflow the blog (Psalm 69:15)

**Review by anguzz — APPROVED** [review]
2 good /ᐠ｡ꞈ｡ᐟ\

**Pull request merged**